### DISCUSSION TITLE
set sleep percentage based on  the number of players online

### DISCUSSION THREAD

**Discussion by Jetpackdan** [nkomarn/Harbor#82] [Ideas]
Hello, I am a player on a server that uses your plugin. 

and I think this future would be helpful 
could it be possible to set the percentage of people needed to sleep based on the total player count 
for an example 

If there are 2 people online, only 50% of players need to sleep.
If there are 3 people online, only 60% of players need to sleep.
If there are 100 people online, only 25% of players need to sleep.

I think this would be a helpful feature for this server plugin.

**Comment by nkomarn** [comment]
Interesting, I'll have a think about how to go about implementing that.

**Reply by Jetpackdan** [reply]
Wow, thanks for the reply, and so soon cant wait to hear if you can implement that future as I have not seen any other sleep plugin with something like that.
